code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -125,29 +122,51 @@
 	name = "kelotane autoinjector"
 	desc = "An auto-injector loaded with 3 doses of kelotane, a common burn medicine."
 	icon_state = "autoinjector-5"
-	amount_per_transfer_from_this = 5
-	list_reagents = list(/datum/reagent/medicine/kelotane = 15)
+	list_reagents = list(/datum/reagent/medicine/kelotane = 30)
 
 /obj/item/reagent_containers/hypospray/autoinjector/bicaridine
 	name = "bicaridine autoinjector"
 	desc = "An auto-injector loaded with 3 doses of bicaridine, a common brute and circulatory damage medicine."
 	icon_state = "autoinjector-3"
-	amount_per_transfer_from_this = 5
-	list_reagents = list(/datum/reagent/medicine/bicaridine = 15)
+	list_reagents = list(/datum/reagent/medicine/bicaridine = 30)
 
 /obj/item/reagent_containers/hypospray/autoinjector/inaprovaline
 	name = "inaprovaline autoinjector"
-	desc = "An auto-injector loaded with 15 units of inaprovaline, an emergency stabilization medicine for patients in critical condition."
+	desc = "An auto-injector loaded with 2 doses of inaprovaline, an emergency stabilization medicine for patients in critical condition."
+	icon_state = "autoinjector-9"
+	amount_per_transfer_from_this = 15
+	list_reagents = list(/datum/reagent/medicine/inaprovaline = 30)
+
+/obj/item/reagent_containers/hypospray/autoinjector/dexalin
+	name = "inaprovaline autoinjector"
+	desc = "An auto-injector loaded with 3 doses of inaprovaline, an emergency stabilization medicine for patients in critical condition."
 	icon_state = "autoinjector-9"
-	list_reagents = list(/datum/reagent/medicine/inaprovaline = 15)
+	volume = 15
+	amount_per_transfer_from_this = 5
+	list_reagents = list(/datum/reagent/medicine/dexalin = 15)
 
 /obj/item/reagent_containers/hypospray/autoinjector/spaceacillin
 	name = "spaceacillin autoinjector"
 	desc = "An auto-injector loaded with 3 doses of spaceacillin, an emergency medicine that help combat against infection before necrosis appears."
 	icon_state = "autoinjector-1"
+	volume = 15
 	amount_per_transfer_from_this = 5
 	list_reagents = list(/datum/reagent/medicine/spaceacillin = 15)
 
+/obj/item/reagent_containers/hypospray/autoinjector/alkysine
+	name = "alkysine autoinjector"
+	desc = "An auto-injector loaded with 3 doses of alkysine, long-living medicine for fixing brain damage."
+	icon_state = "autoinjector-1"
+	volume = 15
+	amount_per_transfer_from_this = 5
+	list_reagents = list(/datum/reagent/medicine/alkysine = 15)
+
+/obj/item/reagent_containers/hypospray/autoinjector/imidazoline
+	name = "imidazoline autoinjector"
+	desc = "An auto-injector loaded with 3 doses of imidazoline, medicine for fixing eyesight."
+	icon_state = "autoinjector-1"
+	list_reagents = list(/datum/reagent/medicine/imidazoline = 30)
+
 /obj/item/reagent_containers/hypospray/autoinjector/hypervene
```

Comment:
Reason for no change to hypervene autoinjector?

Reply:
I believe hypervene autoinjector is meant to be a lower unit amount since you only need about 1-2 units at any time. Makes it need use the autoinjector mechanics instead of being a free 30 units of hypervene in a space you will never use for anything else I guess.

Reply:
> Reason for no change to hypervene autoinjector?

3u is OD for hypervene. Pills also have 3u inside. 3 does are in line with others so looks fine to me.
Thanks for code review above!